src/Notification/Job/SendNotificationsJob.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Flarum\Notification\Job;
+
+use Carbon\Carbon;
+use Flarum\Notification\Blueprint\BlueprintInterface;
+use Flarum\Notification\Event\Notifying;
+use Flarum\Notification\Event\Sending;
+use Flarum\Notification\MailableInterface;
+use Flarum\Notification\Notification;
+use Flarum\Notification\NotificationMailer;
+use Flarum\Notification\NotificationSyncer;
+use Flarum\Queue\AbstractJob;
+use Flarum\User\User;
+
+class SendNotificationsJob extends AbstractJob
+{
+    /**
+     * @var BlueprintInterface
+     */
+    private $blueprint;
+    /**
+     * @var array
+     */
+    private $recipientIds;
+
+    public function __construct(BlueprintInterface $blueprint, array $recipientIds = [])
+    {
+        $this->blueprint = $blueprint;
+        $this->recipientIds = $recipientIds;
+    }
+
+    public function handle(NotificationMailer $mailer)
```

Comment:
Why did you decide to auto-inject this parameter in the `handle()` method and not in the constructor?

Reply:
Jobs never get dependencies inject with Laravel. Construct arguments are for job related payload, see https://laravel.com/docs/5.7/queues#class-structure

Reply:
Right, my bad.